circleci/android-sdk/README.md

```diff
@@ -0,0 +1,115 @@
+# android-sdk Orb
+
+See the [android-sdk Orb page](https://circleci.com/orbs/registry/orb/rakutentech/android-sdk) for details on the jobs included by this Orb.
+
+## Basic Setup
+
+This basic setup will do the following:
+- Checkout the Git repo and initialize any Git submodules.
+- Run the `build` job.
+    - Run Gradle's `check` and `assemble` commands.
+    - Upload coverage to (Code Cov)(https://codecov.io).
+    - Store artifacts: `reports`, `test-results`, `aar`, and `apk`
+    - Persists to workspace: the `build` folder for `sdk-path` and the `apk` folder for the `sample-app-path`.
+- If this is a release (a Git tag that starts with `v`), then pause for user verification.
+- If this is a release (a Git tag that starts with `v`), then run the `publish` job.
```

Comment:
[INQ] Should this be "If verified by user"?

Reply:
Yeah, probably that's better.